types/pubnub/index.d.ts

```diff
@@ -468,7 +468,7 @@ declare namespace Pubnub {
         entry: any;
         timetoken?: string | number;
         meta?: {
-            [key: string]: string;
+            [key: string]: any;
```

Comment:
Hey! Thank you for the change. I was just having a look into the flow annotation: 

https://github.com/pubnub/javascript/blob/master/src/core/flow_interfaces.js#L246-L250

Shall we follow and make it ``meta?: object``?

Reply:
Yeah that seems reasonable to me